darshan-util/pydarshan/setup.py

```diff
@@ -71,11 +71,12 @@
     include_package_data=True,
     keywords="darshan",
     name="darshan",
-    packages=find_packages(include=["darshan*"]),
+    packages=find_packages(include=["darshan*", "examples*", "tests"]),
     setup_requires=setup_requirements,
     test_suite="tests",
     tests_require=test_requirements,
     url='https://www.mcs.anl.gov/research/projects/darshan/',
     version='3.3.0.3',
     zip_safe=False,
+    package_data={"": ["*.darshan"]},
```

Comment:
current release manager may want to take note of consequences to release process--I believe we may end up with larger packages as a result of making the data files broadly available in installs like this